export const useSetError = vi.fn();
export const useSetErrorMock = useSetError;
